test/integration.sh

```diff
@@ -25,7 +25,8 @@ source "${KUBE_ROOT}/hack/lib/init.sh"
 runTests() {
   kube::etcd::start
 
-  go test -race -v github.com/kubernetes-incubator/service-catalog/test/integration/... --args -v 10 -logtostderr
+  go test -race -i github.com/kubernetes-incubator/service-catalog/test/integration/... -c \
```

Comment:
did you mean to commit this?

Reply:
ya, helps it compile faster. go doesn't allow you to both save the build results and use them in the same command, go figure.